doc/developer-guide/internal-libraries/RPC.en.rst

```diff
@@ -0,0 +1,403 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+.. include:: ../../common.defs
+
+.. highlight:: cpp
+.. default-domain:: cpp
+
+.. _developer-doc-rpc:
+
+.. |TServer| replace:: :program:`traffic_server`
```

Comment:
I see there's a |TS| replace. Should I be using this instead? If so, does `traffic_manager` need to look like Traffic Manager for consistency.

Reply:
No, because `|TS|` is for the generic project name, but these are specific executables and should be marked as such.